src/lints/static_unsafe_added.ron

```diff
@@ -0,0 +1,51 @@
+SemverQuery(
+    id: "static_unsafe_added",
+    human_readable_name: "pub static became unsafe",
+    description: "a extern static became unsafe to call.",
```

Comment:
"Call" doesn't seem like the right verb for statics, since they aren't callable. Let's go with "use" instead:
```suggestion
    description: "An extern static became unsafe to use.",
```